optaweb-vehicle-routing-docs/src/main/asciidoc/run-openshift.adoc

```diff
@@ -0,0 +1,79 @@
+[[run-openshift]]
+= Run on OpenShift
+
+Linux and macOS users can use a Bash script called `runOnOpenShift.sh` to install OptaWeb Vehicle Routing on OpenShift.
+
+== Running on a local OpenShift cluster
+
+Use https://developers.redhat.com/products/codeready-containers[Red Hat CodeReady Containers]
+to easily set up a single-node OpenShift 4 cluster on your local computer.
+
+.Prerequisites
+You have successfully built the project with Maven.
+
+.Procedure
+1. To install CRC, follow the link:https://code-ready.github.io/crc/[Red Hat CodeReady Containers Getting Started Guide].
+
+2. When the cluster has started,
```

Comment:
When the cluster starts, perform the following steps:
.. Add the oc (code)...
(the double .. give you the second level list if under an order list (.).